public/index.ts

```diff
@@ -28,7 +31,7 @@ const myState = {
   exposure: 0.75,
   aperture: 0.0,
   fov: 20,
-  focal_distance: 4.0,
+  focalDistance: 4.0,
```

Comment:
@toloudis I hope this wasn't one of those snake case names that depend on json data

Reply:
all the names of properties in the state object of the test app should definitely not be part of any exported interfaces.  should all be local to that app only.